sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -40,6 +41,10 @@ def some_op():
       command=['sleep 1'],
   )
 
+_TEST_GCS_DOWNLOAD_COMPONENT_URL = 'https://raw.githubusercontent.com/kubeflow/'\
+                                   'pipelines/master/components/google-cloud/'\
```

Comment:
Maybe replace `master` with `2dac60c` in case we rename something later.

Reply:
Done.